### ISSUE TITLE
Filtered dropdown for "Pull"

### ISSUE THREAD

**Issue by clounie** [ForkIssues/Tracker#70]
When you're trying to select a branch in the "Pull" dialog and have 30+ branches to look for, it can be a bit of a pain. Being able to narrow it down with some kind of basic text filtering would be awesome.

**Comment by DanPristupov** [comment]
@clounie this is a great idea! Such filter can also be applied to some other dialogs. I'm going to implement as soon as I have free time.

**Comment by bdgeyter** [comment]
+1 on this one. Would be very useful on large projects!

**Comment by jvincent-siplay** [comment]
Yes, this is sorely needed.  Many of our repos have hundreds of branches because most devs at my company don't delete their old branches.  Github separates active vs. stale branches (those without commits in the last 3 months).  It would be awesome if Fork had an option to hide stale branches.
